harness/determined/cli/resource_pool.py

```diff
@@ -77,6 +80,43 @@ def list_workspaces(args: Namespace) -> None:
     return
 
 
+@authentication.required
+def list_resource_pools(args: Namespace) -> None:
```

Comment:
Thanks for putting the functionality into the SDK, first. Two endpoints for the price of one! (well, maybe costs slightly more than one...)